### PR TITLE
💄900 - Add tooltips on buttons

### PR DESCRIPTION
**Pull request by Aibono1225**
- [x] Recommended extensions have been installed if using VS Code

This is to solve the issue on https://github.com/SSWConsulting/SSW.Rules/issues/900

Changes I made:
- Added tooltip component
- Added tooltip to the "+" - "Add new rule"
- Added tooltip to the "?" - "How to edit rules"
- Fixed style problem related to "Edit in GitHub"
image
image

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1642,6 +1641,31 @@ div.bad-example {
...
+  font-size: small;
+  bottom: 80%;
+  position: absolute;
+  opacity: 0;
+  transition: opacity 0.2s;
+  white-space: nowrap;
+  padding: 5px;
+  margin: 0 auto;
+}
+
```
Comment (BrookJeynes) [inline src/style.css:1668]: Is it possible to use tailwind instead?

File: src/style.css [hunk 2]
```diff
@@ -1698,8 +1722,19 @@ div.bad-example {
...
+  content: "";
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translateX(-50%);
+  border-width: 5px;
+  border-style: solid;
+  border-color: #9e9e9e transparent transparent transparent;
+}
+
```
Comment (BrookJeynes) [inline src/style.css:1735]: Same here

**Review by BrookJeynes — CHANGES_REQUESTED** [review]
Logic looks good, just check if the stylings can be done with tailwind :)

File: src/style.css (hunk 2 above)
Comment (Aibono1225) [reply src/style.css]: Done - I replaced the styling with Tailwind 😊 also added linter for better tailwind

**Review by BrookJeynes — APPROVED** [review]

**Pull request merged**